modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -3,26 +3,46 @@
 // installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
 :_content-type: PROCEDURE
-[id="configuring-host-network-interfaces-in-the-install-config-yaml-file_{context}"]
-= (Optional) Configuring host network interfaces
+[id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
+= Optional: Configuring host network interfaces in the `install-config.yaml` file
 
-During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration.
+During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState.
 
 [IMPORTANT]
 ====
-Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes.
+Host network interface configuration is set during the installation and provisioning process and includes advanced options such as setting static IPs per host. Before configuring host network interfaces, review the {product-title} 4.10 release notes.
 ====
 
-See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
+.Prerequisites
 
-.Example
-[source,yaml]
+* Configure a `PTR` DNS record with a valid hostname for each node with a static IP address.
+
+* Install the `jq` package.
+
+.Procedure
+
+. Save the valid `networkConfig` YAML syntax to a file.
+. Generate the configuration file for the `networkConfig` YAML file. Run the following command:
++
+[source,terminal]
+----
+$ nmstatectl gc <networkConfig-configuration-file> 
+----
++
+[NOTE]
+====
+Consider testing the `networkConfig` YAML syntax before including it in the `install-config.yaml` file. Errors in the YAML syntax might result in a failure to apply the network configuration. Maintain the valid YAML syntax when applying changes using Kubernetes NMState after deployment or when expanding the cluster.
```

Comment:
Looks like this note was moved a bit too far down. To implement this suggestion, the user would have to run the 'nmstatectl' command that's currently in line 29.

Reply:
All of the instructions can be moved below the example listed below.